manifests/y/Yubico/Authenticator/5.1.0/Yubico.Authenticator.locale.en-US.yaml

```diff
@@ -1,25 +1,26 @@
-# Created using wingetcreate 0.4.1.1
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
 
 PackageIdentifier: Yubico.Authenticator
 PackageVersion: 5.1.0
 PackageLocale: en-US
 Publisher: Yubico AB
 PublisherUrl: https://www.yubico.com/
-# PublisherSupportUrl: https://support.yubico.com
-
+PublisherSupportUrl: https://support.yubico.com

```

Comment:
The reason for the commented out support url was because the support URL started blocking the validation's IP address (Error Code: Forbidden)
We'll see how long we're allowed to access that link

Reply:
Ah. I wasn't sure and it was a working link. We can comment it again if that's best.

Reply:
naa, it will fail on the next PR if thats the case, i don't think wingetbot regularly scan the urls in the manifest (its not very good at it if it does)